sunpy/database/database.py

```diff
@@ -808,7 +808,7 @@ def download_from_hek_query_result(self, query_result, client=None, path=None, p
 
         Parameters
         ----------
-        query_result : `HEKTable` or `HEKRow`
+        query_result : `HEKResponse` or `HEKRow`
```

Comment:
these should be fully  qualified so links work?

Reply:
Fixed.